services/rds/rds.go

```diff
@@ -149,6 +151,18 @@ func (svc *Service) ApplyPrometheusConfiguration(ctx context.Context, q *reform.
 			Replacement: "mysql",
 		}},
 	}
+	rdsBasic := &prometheus.ScrapeConfig{
+		JobName:        "rds-basic",
+		ScrapeInterval: "60s",
+		ScrapeTimeout:  "55s",
+		MetricsPath:    "/basic",
+	}
+	rdsEnhanced := &prometheus.ScrapeConfig{
+		JobName:        "rds-enhanced",
+		ScrapeInterval: "10s",
+		ScrapeTimeout:  "9s",
+		MetricsPath:    "/enhanced",
+	}
```

Comment:
Shouldn't this be in some configuration file?

Reply:
An interval for basic metrics is fixed by AWS, so no configuration option is necessary.
Interval for enhanced metrics should be auto-adjustable, but that's another issue.